tests/test_initialize_simulation.py

```diff
@@ -79,3 +84,33 @@ def test_initialize_simulation_with_sampling(self):
         self.assertEqual(len(sim.vehicles), 20, "should have loaded 20 vehicles")
         self.assertEqual(len(sim.stations), 4, "should have loaded 4 stations")
         self.assertEqual(len(sim.bases), 2, "should have loaded 2 bases")
+
+    def test_initialize_simulation_load_from_geofence_file(self):
```

Comment:
i appreciate the idea of testing this, but, this implies every time we test hive that we are making a call to OpenStreetMaps. we probably shouldn't do this unless we can cache the query result locally for osmnx, which may not be worth our time to figure out. perhaps we flag this test with `@unittest.skip("makes API call to OpenStreetMaps via osmnx")`?

Reply:
Agreed, added